### PR TITLE
Issues with section syncs etc

### PR DESCRIPTION
**Pull request by Boy0000**

### CODE REVIEW

**Comment by Boy0000** [comment]
Might be worth doing a config thing for the ladder disabling. but it works for now.
Since FAWE wont work on survival it will be a pain to remove them.

File: src/main/java/com/derongan/minecraft/deeperworld/synchronization/SectionSyncListener.kt [hunk 1]
```diff
@@ -95,6 +98,16 @@ object SectionSyncListener : Listener {
         }
     }
 
+    @EventHandler
+    fun BlockPlaceEvent.disableBlocks() {
+        val section = WorldManager.getSectionFor(player.location) ?: return
```
Comment (0ffz) [inline src/main/java/com/derongan/minecraft/deeperworld/synchronization/SectionSyncListener.kt:103]: Plz add a toggle for this in the config that is off by default.

**Review by 0ffz — CHANGES_REQUESTED** [review]
Looks good, just the 1 comment

File: src/main/java/com/derongan/minecraft/deeperworld/synchronization/SectionSyncListener.kt (hunk 1 above)
Comment (Norazan) [reply src/main/java/com/derongan/minecraft/deeperworld/synchronization/SectionSyncListener.kt:103]: this function can be removed once my changes in MadeInAbyss go through tmorrow

**Comment by Boy0000** [comment]
@Norazan moved this properly over to MineInAbyss

**Pull request closed without merging**